src/base-command.ts

```diff
@@ -66,14 +72,21 @@ export abstract class BaseCommand<T extends typeof Command> extends Command {
         process.env.TENANT_ID = accessibleTenants.data[0].id
         this.log(ux.colorize('yellow', `✓ Found single accessible Tenant. Using ${process.env.TENANT_ID}.`))
       } else {
+        this.log(
+          ux.colorize(
+            'blueBright',
+            `Your credentials can access tenants:\n${accessibleTenants.data.map((x) => `- ${x.attributes.name}: ${x.id}`).join(',\n')}.\n`,
+          ),
+        )
         this.log(
           ux.colorize(
             'yellow',
-            `Your credentials can access tenants ${accessibleTenants.data.map((x) => `[${x.attributes.name}:${x.id}]`).join(',')}. Please set TENANT_ID for the tenant of your choice.`,
+            `export TENANT_ID=${process.env.TENANT_ID} (Windows: set TENANT_ID=${process.env.TENANT_ID}) to avoid inputting this for each command.\n`,
```

Comment:
q: might we want to write non-sensitive data to a local config file, similar to how the Snyk CLI does this under `~/.config/configstore/broker-config.json`?